### DISCUSSION TITLE
Country Ranking does not update for a long time

### DISCUSSION THREAD

**Discussion by Shirou27** [ppy/osu#27846] [Ideas] [answered]
I've been facing this problem for a long time. My country ranking is not updating, it's just a blank. I hope this will be fix. I'm using an android phone by the way.

![Screenshot_2024-04-13-21-58-37-108-edit_com opera browser](https://github.com/ppy/osu/assets/105548541/fcc38dc2-3d22-4dd8-bdc9-a2b895103a5f)

**Comment by Joehuu** [comment]
Should probably be opened on `osu-web`.

**Reply by Walavouchey** [reply, marked as answer]
https://github.com/ppy/osu-web/issues/11158
